### PR TITLE
#43: Update on top of 0.6.4

### LINKED ISSUE #43: Update on top of 0.6.4
## Situation

There is a version collision between the BucketFS library used in this project and `exasol-test-setup-abstraction-java` that leads to a class-not-found error in certain combination. An update to the latest versions will fix this.

## Acceptance Criteria

* Dependencies updated to latest.
* Tests green

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #43.

### CODE REVIEW

File: doc/changes/changes_0.6.5.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# udf-debugging-java 0.6.5, released 2022-11-22
+
+Code name: Updated dependencies on top of 0.6.4
+
+## Summary
+
+In this release we fixed a version collision between the BucketFS library used in this project and `exasol-test-setup-abstraction-java` that lead to a class-not-found error in certain combinations.
```
Comment (ckunki) [inline doc/changes/changes_0.6.5.md:7]:
```suggestion
In this release we fixed a version collision between the BucketFS library used in this project and `exasol-test-setup-abstraction-java` that led to a class-not-found error in certain combinations.
```

File: doc/changes/changes_0.6.5.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Known Issues
+
+This project depends on an Amazon AWS SDK which in turn depends on the Netty HTTP server version 4.1.77. This versions has a vulnerability in certificate validation that can lead to man-in-the-middle attacks. Unfortunately, no update of the AWS SDK is available at the time of this release.
```
Comment (ckunki) [inline doc/changes/changes_0.6.5.md:11]:
```suggestion
This project depends on an Amazon AWS SDK which in turn depends on the Netty HTTP server version 4.1.77. This versions has a vulnerability in certificate validation that can allow to man-in-the-middle attacks. Unfortunately, no update of the AWS SDK is available at the time of this release.
```

**Review by ckunki — CHANGES_REQUESTED** [review]

**Review by ckunki — APPROVED** [review]

**Review by ckunki — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
     <artifactId>udf-debugging-java</artifactId>
-    <version>0.6.4</version>
+    <version>0.6.5</version>
     <name>udf-debugging-java</name>
     <description>Utilities for debugging, profiling and code coverage measure for UDFs.</description>
     <url>https://github.com/exasol/udf-debugging-java/</url>
     <properties>
-        <junit.version>5.8.2</junit.version>
+        <junit.version>5.9.1</junit.version>
         <vscommon.version>11.0.0</vscommon.version>
```
Comment (ckunki) [inline pom.xml:11]:
```suggestion
        <vscommon.version>11.0.0</vscommon.version>
        <jacoco.version>0.8.8</jacoco.version>
```

Comment (ckunki) [reply pom.xml:11]: Required by later dependencies.

**Review by ckunki — CHANGES_REQUESTED** [review]

**Comment by redcatbear** [comment]
Looks like I was a little bit overeager in my cleaning up. Restored jacoco property.

**Review by ckunki — APPROVED** [review]

**Pull request merged**